### DISCUSSION TITLE
2024.3.4

### DISCUSSION THREAD

**Discussion by AndrewBastin** [hoppscotch/hoppscotch#4128] [Announcements]
This release introduces minor improvements and fixes for Selfhosting, some fixes for OAuth flows and support for headers having same key with multiple value in responses when using the Desktop App.

## What's Changed
* HSB-431 fix: email case sensitive in email provider by @mirarifhasan in https://github.com/hoppscotch/hoppscotch/pull/4042
* fix: persist interceptor state for logged out user by @nivedin in https://github.com/hoppscotch/hoppscotch/pull/4060
* fix: add previous value as optional in test schema env diff by @nivedin in https://github.com/hoppscotch/hoppscotch/pull/4071
* fix: precision lost when json is beautified by @nivedin in https://github.com/hoppscotch/hoppscotch/pull/4086
* fix(sh-admin): saving changes to server configurations post a failed attempt will require a page reload in dashboard by @JoelJacobStephen in https://github.com/hoppscotch/hoppscotch/pull/4081
* locale: update tr i18n translations by @islamzeki in https://github.com/hoppscotch/hoppscotch/pull/4057
* refactor(sh-admin): improved error handling and dynamic user actions in admin dashboard by @JoelJacobStephen in https://github.com/hoppscotch/hoppscotch/pull/4044
* fix: make client credential optional in authcode + pkce flow by @amk-dev in https://github.com/hoppscotch/hoppscotch/pull/4114
* fix: headers having different values with the same key are not shown by @AndrewBastin in https://github.com/hoppscotch/hoppscotch/pull/4112

**Full Changelog**: https://github.com/hoppscotch/hoppscotch/compare/2024.3.3...2024.3.4

<hr /><em>This discussion was created from the release <a href='https://github.com/hoppscotch/hoppscotch/releases/tag/2024.3.4'>2024.3.4</a>.</em>

**Comment by baek-sang** [comment]
<img width="639" alt="new" src="https://github.com/hoppscotch/hoppscotch/assets/69100154/c60a3a6f-bda5-49d9-8773-f060595a6b3e">

Any news on the desktop app?
